settings/serverSettings.ts

```diff
@@ -152,6 +152,9 @@ export const GDOCS_HOMEPAGE_CONFIG_DOCUMENT_ID: string =
 
 export const GDOCS_SHARED_DRIVE_ID = serverSettings.GDOCS_SHARED_DRIVE_ID ?? ""
 
+export const GDOCS_DETAILS_ON_DEMAND_ID =
```

Comment:
TODO: add error handling to help people if this variable is unset

Reply:
Ideally, our site would work "relatively" fine even if this setting is not set.

Reply:
Hopefully this time I'll have learned my lesson from all the people needing help in slack with the last round of Gdocs variables 😅